### PR TITLE
Feat/change default redirection

### PR DESCRIPTION
**Pull request by zatteo**
```
### ✨ Features

* Change your default app in profile section
```

Related : 
- https://github.com/cozy/cozy-libs/pull/2069
- https://github.com/cozy/cozy-flagship-app/pull/682

### CODE REVIEW

File: src/components/DefaultRedirectionSection.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+const formatOptions = (appsResult, t) => {
+  if (!hasQueryBeenLoaded(appsResult)) return []
+
+  return appsResult.data
+    .filter(({ slug }) => !EXCLUDED_SLUGS.includes(slug))
+    .map(({ slug, name }, index) => {
+      if (slug === 'home') {
+        return {
+          index: -1, // to set 'home' before everything else when sorting
```
Comment (Ldoppea) [inline src/components/DefaultRedirectionSection.jsx:24]: Two questions:
- Should we move this logic in the `.sort()` instead? Using a .map to do a sort is a bit odd
- Don't we sort other apps alphabetically?

Comment (Ldoppea) [reply src/components/DefaultRedirectionSection.jsx:24]: After reading a bit more, I understand that we do the map in order to rename the `home` option, so it is more optimal to also do the index thing here.
But the question still remain about the alpabetical sort

File: src/components/DefaultRedirectionSection.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            value: app.slug,
+            label: app.name
+          }
+        })}
+        fieldProps={fieldProps}
+        value={selectedSlug}
+        onChange={sel => {
+          const newDefaultRedirection = formatDefaultRedirection(sel.value)
+          onChange(fieldName, newDefaultRedirection)
+          webviewIntent.call('setDefaultRedirection', newDefaultRedirection)
```
Comment (Ldoppea) [inline src/components/DefaultRedirectionSection.jsx:93]: Should we condition this call to `isFlagshipApp()`?

Comment (zatteo) [reply src/components/DefaultRedirectionSection.jsx:93]: Yes I think

File: src/components/DefaultRedirectionSection.jsx (hunk 1 above)
Comment (zatteo) [reply src/components/DefaultRedirectionSection.jsx:24]: When I checked they are returned alphabetically sorted by the stack so I was counting on it.

Comment (Ldoppea) [reply src/components/DefaultRedirectionSection.jsx]: Ok.

@Crash-- do you know if this is a "lucky" effect or if we can consider this to be the norm?

However this is a detail that should not block the PR so I will approve.

**Review by Ldoppea — APPROVED** [review]

**Pull request merged**